.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,29 +1,51 @@
 ---
 name: Bug report
 about: Create a report to help us improve the library
+title: ''
+labels: ''
+assignees: ''
+
 ---
 
+<!-- Before raising, please check if somebody else has already reported your issue. -->
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Steps to reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
 
-### Description
-<!-- A quick description of what you're trying to accomplish -->
+### Environment
+* Bugsnag version:
+* Unity version:
```

Comment:
I'd suggest the following information from [player settings](https://docs.unity3d.com/2018.2/Documentation/Manual/class-PlayerSettings.html) would be useful to collect too:

- Scripting backend (Mono or IL2CPP)
- API compatibility level for .NET
- Stack Trace level for all error types (None/ScriptOnly/Full)

Reply:
Thanks, added.